packages/html-spec/src/forms/forms.json

```diff
@@ -0,0 +1,82 @@
+[
+    {
+        "component": "forms",
+        "selector": [
+            ".k-form"
+        ],
+        "states": [
+            {
+                "name": "error",
+                "selector": [
+                    ".k-form-error"
+                ]
+            }
+        ],
+        "options": [
+            {
+                "name": "size",
+                "values": [
+                    {
+                        "name": "small",
+                        "selector": [
+                            ".k-form-sm"
+                        ]
+                    },
+                    {
+                        "name": "medium",
+                        "selector": [
+                            ".k-form-md"
+                        ]
+                    },
+                    {
+                        "name": "large",
+                        "selector": [
+                            ".k-form-lg"
+                        ]
+                    }
+                ]
+            }
+        ],
+        "children": [
+            {
+                "name": "form-fieldset",
+                "selector": [
+                    ".k-form-fieldset"
+                ]
+            },
+            {
+                "name": "form-field",
+                "$ref": "form-field",
+                "selector": [
+                    ".k-form-field"
+                ]
+            },
+            {
+                "name": "form-buttons",
+                "selector": [
+                    ".k-form-buttons"
+                ]
+            }
+        ]
+    },
+    {
+        "component": "form-field",
+        "selector": [
+            ".k-form-field"
+        ],
+        "children": [
```

Comment:
The form hint `.k-form-hint` element is not described. Please add it to the list of children.